src/apiClient.js

```diff
@@ -743,22 +751,66 @@ class ApiClient {
     }
 
     getDownloadSpeed(byteSize) {
-        const url = this.getUrl('Playback/BitrateTest', {
-            Size: byteSize
-        });
+        return new Promise((resolve, reject) => {
+            const url = this.getUrl('Playback/BitrateTest', {
+                Size: byteSize
+            });
 
-        const now = new Date().getTime();
+            console.log(`Requesting ${url}`);
```

Comment:
Is this log something that should be kept?

Reply:
AFAIK, we print every request. It was printed by `ajax` (fetch wrapper).